presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkQueryExecutionFactory.java

```diff
@@ -149,6 +149,7 @@
 import static com.facebook.presto.spark.util.PrestoSparkUtils.getActionResultWithTimeout;
 import static com.facebook.presto.spi.StandardErrorCode.GENERIC_INTERNAL_ERROR;
 import static com.facebook.presto.spi.StandardErrorCode.INVALID_SESSION_PROPERTY;
+import static com.facebook.presto.sql.QueryUtil.isRollBack;
```

Comment:
This couples to a new package & module. Probably not worth it for the trivial method you're importing.

Reply:
Thanks for the comment, I also struggled here for a while. Fixed as your suggestion.